tests/test_csc.py

```diff
@@ -214,8 +214,9 @@ def mock_getList(a_listSize, a_pRequiredSize, a_pList):
                 duration = 1e-9  # seconds
                 with salobj.assertRaisesAckError(ack=salobj.SalRetCode.CMD_FAILED,
                                                  result_contains="Exposure duration"):
-                    await harness.remote.cmd_expose.set_start(timeout=STD_TIMEOUT+duration, duration=duration)
-
+                    timeout = STD_TIMEOUT + duration
+                    await asyncio.create_task(harness.remote.cmd_expose.set_start(timeout=timeout,
+                                                                                  duration=duration))
```

Comment:
Why not use STD_TIMEOUT for this case? The command should be rejected immediately, not after `duration` seconds.

Reply:
Good point: you're absolutely right.